src/django_components/util/tag_parser.py

```diff
@@ -1,37 +1,71 @@
 from dataclasses import dataclass
 from typing import List, Optional, Sequence, Tuple, Union
 
-TAG_WHITESPACE = (" ", "\t", "\n", "\r", "\f")
+TAG_WHITESPACES = (" ", "\t", "\n", "\r", "\f")
+TAG_QUOTES = ("'", '"', '_("', "_('")
+TAG_FILTER_JOINERS = ("|", ":")
+TAG_SPREAD = "..."
 
 
 @dataclass
-class TagAttr:
-    key: Optional[str]
+class TagAttrPart:
```

Comment:
Previously I considered the cases of args (`"abc"`), kwargs (`key=value`), spread (`...my_attrs`) and translation (`_("my_text")`). But I didn't consider that values can have filters with `|`, and filter can have args with `:`.

Before this PR, the implementation expected that the value can be only EITHER string literal like `"abc"`, or a variable / value like `my_var`.

But with filters, both of these can be present simultaneously, e.g. `key="MY_TEXT"|lower` or `key=my_var|yesno:"1,2"`.

So the parser now recognizes the "parts".
- In `key="MY_TEXT"|lower`, the parts are text `"MY_TEXT"` and filter `lower`
- In `key=my_var|yesno:"1,2"`, the parts are variable `my_var`, filter `yesno`, and filter arg text `"1,2"`